Examples/Io/Csv/src/CsvSpacepointsWriter.cpp

```diff
@@ -0,0 +1,58 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/Io/Csv/CsvSpacepointsWriter.hpp"
+#include "ActsExamples/EventData/SimSpacePoint.hpp"
+#include "Acts/Definitions/Units.hpp"
+
+#include "ActsExamples/Utilities/Paths.hpp"
+
+#include <ios>
+#include <optional>
+#include <stdexcept>
+
+#include <dfe/dfe_io_dsv.hpp>
+
+#include "CsvOutputData.hpp"
+
+ActsExamples::CsvSpacepointsWriter::CsvSpacepointsWriter(
+    const ActsExamples::CsvSpacepointsWriter::Config& cfg,
+    Acts::Logging::Level lvl)
```

Comment:
For reasons related to #845, could you rename the arguments to?
```suggestion
    const ActsExamples::CsvSpacepointsWriter::Config& config,
    Acts::Logging::Level level)
```